sunpy/map/maputils.py

```diff
@@ -1,7 +1,309 @@
+          keyword_test  is not in the accepted meta data required for sunpy.map.Map 
+    
+    Creating headers from map_helper:
+
+        * Create a default header with meta data assumed to be Earth-based observer at current time and arcsec units
+          >>> from sunpy import map
+          >>> from sunpy.coordinates import frames
+          >>> from astropy.coordinates import SkyCoord
+          >>> from astropy import units as u
+          >>> import numpy as np
+
+          >>> data = np.random.rand(1024, 1024)
+          >>> header = map.MapHelper.make_header(data)
+          >>> my_map = map.Map(data, header)
+        
+        * Create a header from a reference SkyCoord object
+          >>> my_coord = SkyCoord(100*u.arcsec, 100*u.arcsec, obstime = '2013-10-28 01:00', frame = frames.Helioprojective)
+          >>> header = map.MapHelper.make_header(data, my_coord)
+          >>> my_map = map.Map(data, header)
+
+
+
+    """
+    def acceptable_meta_keywords(self, *key):
+        """
+        prints the accepted metadata keywords that are used when
+        creating a `sunpy.map.Map`.
+        """
+        if key:
+            for k in key:
+                if k in acceptable_meta.keys():
+                    print(k, ' : ', acceptable_meta[k])
+                else:
+                    print(k, ' is not in the accepted meta data required for sunpy.map.Map \n The accepted keywords are : \n', list(acceptable_meta.keys()))
+            return
+
+        for k in acceptable_meta:
+            print(k, ' : ', acceptable_meta[k])
+
+    def make_header(self, data, *coordinate, crval=None, crpix=None, cdelt=None, **kwargs):
+        """
+        Function to create a MetaDict from a coordinate object (SkyCoord or coordinate frame)
+        that is required to create a `sunpy.map.Map`
+
+        Parameters
+        ----------
+        data : `~numpy.ndarray`
+            array data of Map for which a header is requried
+
+        coordinates : `~astropy.coordinates.SkyCoord` or `~astropy.coordinates.BaseFrame`, optional if not set default parameters are set
+            coordinate object to get meta information for map header
+
+        crval : array or list of size of 2, default (0, 0)
+            coordinate of reference pixel along each axis
+            first index is crval1, second index is crval2
+
+        crpix : array or list of size of 2, default center of data array ((data.shape[0] + 1)/2., (data.shape[1] + 1)/2.)
+            reference pixel along each axis
+            first index is crpix1, second index is crpix2
+
+        cdelt : array or list of size of 2, default (0, 0)
+            pixel scaling along each axis (i.e. the spatial scale of the pixels)
+            first index is cdelt1, second index is cdelt2
+
+        **kwargs
+            additional arguments that will be put into metadict header. These must be in the
+            list of acceptable meta keywords
+
```

Comment:
I don't see where the `**kwargs`` are passed in. Maybe I'm blind.

Reply:
in the last line in the function 
```for key in kwargs: ```